control.lua

```diff
@@ -188,6 +188,11 @@ end
 local lastLoggedTagCount = {}
 
 local function updateMapTags(surface, force, chunkPosition, resource)
+	-- Ignore SE core seams. SE places it's own tags.
+	if string.find(string.lower(resource), "core") then
```

Comment:
From the other conversation, this sounds like it should be `"se-core-fragment-"` for the literal string.
And should be starts with (instead of contains).